.circleci/config.yml

```diff
@@ -242,7 +242,8 @@ workflows:
 
       # Chart jobs
       - validate-charts
-      - check-version-bump
+      - check-version-bump:
+          context: runner-package-deploy
```

Comment:
Fixes the version bump check. I'll need to cherry-pick this into the server branches